migrate/live-migration/live-migration-faq.md

```diff
@@ -0,0 +1,85 @@
+---
+title: FAQ
+excerpt: Live-migration Frequently Asked Questions
+products: [cloud]
+keywords: [migration, low-downtime, live-migration, FAQ]
+tags: [migration, logical backup, replication]
+---
+
+# Frequently Asked Questions
+
+This document addresses frequently asked questions and common scenarios you may
+encounter during live migration.
+
+
+## Where can I find logs for processes running during live migration?
+
+Live migration involves several background processes to manage different stages of
+the migration. The logs of these processes can be helpful for troubleshooting
+unexpected behavior. You can find these logs in the `<volume_mount>/logs` directory.
+
+
+## Why does pgcopydb log "no tuple identifier" warning?
+
+`pgcopydb follow` logs a warning `WARNING: no tuple identifier for UPDATE in table "xxxx"`
+when it cannot determine which specific rows should be updated after receiving an
+`UPDATE` statement from the source database during replication. This occurs when tables
+in the source database that receive `UPDATE` statements lack either a `PRIMARY KEY` or
+a `REPLICA IDENTITY` setting. For live migration to successfully replicate `UPDATE` and
+`DELETE` statements, tables must have either a `PRIMARY KEY` or `REPLICA IDENTITY` set
+as a prerequisite.
+
+
+## Set REPLICA IDENTITY on PostgreSQL partitioned tables
+
+If your PostgreSQL tables use native partitioning, setting `REPLICA IDENTITY` on the
+root (parent) table will not automatically apply it to the partitioned child tables.
+You must manually set `REPLICA IDENTITY` on each partitioned child table.
+
+
+## Can I use read/failover replicas as source database for live migration?
+
+Live migration does not support replication from read or failover replicas. You must
+provide a connection string that points directly to your source database for
+live migration.
+
+
+## Can I use Timescale Cloud instance as source for live migration?
+
+No, Timescale Cloud cannot be used as a source database for live migration.
+
+
+## Why does pgcopydb log warnings for continuous aggregates?
+
+Continuous aggregates with `materialized_only=True` use an underlying hypertable
+to store materialized aggregates. When pgcopydb receives an `UPDATE` or `DELETE` statement
+for a hypertable without a `REPLICA IDENTITY`, it cannot identify which rows to
+update or delete, resulting in a warning `WARNING: no tuple identifier for DELETE in table "_timescaledb_internal"."_hyper_18_2940_chunk"`.
+To avoid this warning, set `REPLICA IDENTITY FULL` on all hypertables that support
+continuous aggregates. Live migration handles this automatically, but if you are
+manually performing live migration, you can use the following query
```

Comment:
Is it realistic that people will be doing this? I don't think so.

Reply:
No longer needed.